### ISSUE TITLE
Bertabs metrics lower than paper

### ISSUE THREAD

**Issue by kyungyunji** [huggingface/transformers#3647]
I tested abstractive summarization pre-trained model using the source under transformers/examples/summarization/bertabs/...

My dataset are CNN & Daily mail, which are 30 thousands of docs.

However, the result of rouge score is as follows

****** ROUGE SCORES ******

** ROUGE 1
F1        >> 0.275
Precision >> 0.299
Recall    >> 0.260

** ROUGE 2
F1        >> 0.161
Precision >> 0.184
Recall    >> 0.149

** ROUGE L
F1        >> 0.305
Precision >> 0.326
Recall    >> 0.290

why is the result different from that of the article, Text Summarization with Pretrained Encoders ?

**Comment by PetreanuAndi** [comment]
bump. i have the same question. paper says 5 BeamSize and better accuracy for bert-base.

**Comment by alexgaskell10** [comment]
same here
